website/static/js/citationsNodeConfig.js

```diff
@@ -60,6 +62,12 @@ var CitationsFolderPickerViewModel = oop.extend(FolderPickerViewModel, {
                     return item.kind === 'folder';
                     });
                 },
+                ondataloaderror: function(err){
+                    $osf.handleAddonApiHTTPError(err);
+                    self.destroyPicker();
+                    self.loading(false);
+                    $(self.folderpickerSelector).html(errorpage);
```

Comment:
Please use `self.changeMessage()` if you can here, since changing the knockout `self.message` observable is more reliable than injecting html onto the page with jquery.